api/src/pcapi/templates/invoices/debit_note.html

```diff
@@ -0,0 +1,299 @@
+        fill="#870087"/>
+  <rect y="64" width="4" height="531" transform="rotate(-90 0 64)" fill="#870087"/>
+  <defs>
+    <clipPath id="clip0_3070_3816">
+      <rect width="181" height="65" fill="white" transform="translate(382 60)"/>
+    </clipPath>
+  </defs>
+</svg>
+<h1>Note de débit</h1>
+<div class="customerInfo">
+  <p><b>Structure :</b> {{ reimbursement_point.managingOfferer.name }} - {{ reimbursement_point.managingOfferer.siren }}
+  </p>
+  <p><b>Point de remboursement :</b> {{ reimbursement_point_name }} - {{ reimbursement_point_iban }}</p>
+</div>
+<h3 class="coloredTitle">
+  Note de débit suite à la création d’incident de paiement.
```

Comment:
[question] La formulation me semble bizarre. Le terme de "création" pourrait paraître étrange aux non-techniciens. Suggestion plus simple : 

```suggestion
  Note de débit suite à un incident de paiement
```